test/hotspot/jtreg/vmTestbase/nsk/jvmti/SuspendThread/suspendthrd003/suspendthrd003.cpp

```diff
@@ -97,16 +95,15 @@ agentProc(jvmtiEnv* jvmti, JNIEnv* jni, void* arg) {
         }
         /* Original agentProc test block ends here. */
 
-        /*
-         * Using printf() instead of NSK_DISPLAY1() in this loop
-         * in order to slow down the rate of SuspendThread() calls.
```

Comment:
I don't understand the reason for wanting to slow down the rate, nor your reason for removing it.

Reply:
I couldn't remember why I did that when I wrote the test so
I removed it.

Reply:
Maybe to allow the resume to get the thread back into real code before the next suspend hits?